src/main/java/com/mazawrath/beanbot/utilities/LivestreamNotification.java

```diff
@@ -42,4 +42,32 @@ int getViewerCount() {
     String getThumbnail() {
         return thumbnail;
     }
+
+    private String randomPasswordGenerator(int passwordLength) {
+        // A strong password has Cap_chars, Lower_chars,
+        // numeric value and symbols. So we are using all of
+        // them to generate our password
+        String Capital_chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        String Small_chars = "abcdefghijklmnopqrstuvwxyz";
+        String numbers = "0123456789";
+        //String symbols = "!@#$%^&*_=+-/.?<>)";
+
+
+        String values = Capital_chars + Small_chars +
```

Comment:
In a next iteration of the code better make all these constants so they don't get declared/initialized N times during a life cycle.